### DISCUSSION TITLE
Open project is a system command

### DISCUSSION THREAD

**Discussion by AlanBreck** [johnlindquist/kit#507] [Errors] [answered]
I'm attempting to create a script called `Open project`, but being told that it's a system command. I'm not aware of this command existing. Where would this command be coming from?

![Screen Shot 2021-11-05 at 10 03 37 AM](https://user-images.githubusercontent.com/1199820/140522905-2ce129da-b1cc-46a6-b3ee-0634bf4937ab.png)

**Comment by johnlindquist** [comment, marked as answer]
Oh, that's a bit of bug on my end:

- `open` is a system command.
- Script names require dashes: `open-project`

The "new script" validator checks for system commands _before_ checking for dashes, so you're seeing the wrong error message 😅

**Reply by AlanBreck** [reply]
Aha! Thank you again!
